### DISCUSSION TITLE
2011/08/23/labview-web-ui-builder

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#344] [Announcements]
# 2011/08/23/labview-web-ui-builder

https://labview.qizhen.xyz/

https://blog.qizhen.xyz/2011/08/23/labview-web-ui-builder

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Scienve
2015年01月10日 11:07 编辑
这个好像是基于silverlight开发的，安卓设备没法用啊

回复
Pingback： Modkit 一种图形化编程语言 « 凡人琐忆 编辑
nextpad
2011年08月24日 17:21 编辑
great ， finally come

回复
